packages/h5py/patches/configure.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/setup_configure.py b/setup_configure.py
+index 16c355b..2273ae3 100644
+--- a/setup_configure.py
++++ b/setup_configure.py
+@@ -183,7 +183,8 @@ class BuildConfig:
+ class HDF5LibWrapper:
+ 
+     def __init__(self, libdirs):
+-        self._load_hdf5_lib(libdirs)
++        # self._load_hdf5_lib(libdirs)
++        pass
+ 
+     def _load_hdf5_lib(self, libdirs):
+         """
+@@ -248,20 +249,21 @@ class HDF5LibWrapper:
+ 
+         Raises an exception if anything goes wrong.
+         """
+-        import ctypes
+-        from ctypes import byref
++        # import ctypes
++        # from ctypes import byref
```

Comment:
Just delete stuff rather than commenting it out. It makes the patch shorter and the deleted code is in the patch in any case.
```suggestion
```